### ISSUE TITLE
Improve documentation for theming

### ISSUE THREAD

**Issue by the133448** [Semantic-Org/Semantic-UI-React#4173]
Currently under the Theming with Create React App section of https://react.semantic-ui.com/theming in the step `This file is mostly prepared, but you have to change some details in the bottom of it:` It gives an example of what it should be.

I was confused why after copying and pasting the Folders section of this that it was still not working. It is because I also need to change and update the `Import Theme` section inside the theme.confg.

I propose to update the documentation to show that.
```
/*******************************
            Folders
*******************************/

@themesFolder : 'themes';
@siteFolder  : '../../src/semantic-ui/site';

/*******************************
         Import Theme
*******************************/

@import (multiple) 'semantic-ui-less/theme.less';
@fontPath : '../../../themes/@{theme}/assets/fonts';
```

**Comment by wise-introvert** [comment]
@the133448 , can you please also add where exactly the `../../../theme....` part is coming from? I followed the theming guide and execute `$  npx @semantic-ui-react/bootstrap` which created the `src/semantic-ui` folder, however there is no `themes` folder that the `@fontPath...` line seems to be pointing towards.

**Comment by layershifter** [comment]
Was fixed in [`semantic-ui-react@2.0.4`](https://github.com/Semantic-Org/Semantic-UI-React/releases/tag/v2.0.4).
